cpp/tests/io/json_tree.cpp

```diff
@@ -361,9 +421,114 @@ tree_meta_t2 get_tree_representation_cpu(device_span<PdaTokenT const> tokens_gpu
           std::move(node_range_end)};
 }
 
+std::tuple<std::vector<NodeIndexT>, std::vector<size_type>> records_orient_tree_traversal_cpu(
+  host_span<SymbolT const> input, tree_meta_t2 const& tree, rmm::cuda_stream_view stream)
+{
+  std::vector<NodeIndexT> node_ids(tree.parent_node_ids.size());
+  std::iota(node_ids.begin(), node_ids.end(), 0);
+
+  if (std::getenv("CUDA_DBG_DUMP") != nullptr) {
```

Comment:
We are keeping this? Perhaps a more IO/JSON-ish name?

Reply:
replaced with `NJP_DEBUG_DUMP`